Tests/SourceKittenFramework/Fixtures/Commandant.json

```diff
@@ -978,11 +978,11 @@
       },
       {
         "key.kind" : "source.lang.swift.decl.extension",
-        "key.offset" : 326,
+        "key.offset" : 1603,
```

Comment:
This was an example that the offset and length (326,14) locates in different fie.
The value was offset of `NSCharacterSet` in `NSCharacterSet.h`:
![screenshot 2016-06-10 19 18 12](https://cloud.githubusercontent.com/assets/33430/15961591/7f969cdc-2f40-11e6-8d59-a557ccfc2bfc.png)